pallet-logion-loc/src/lib.rs

```diff
@@ -1222,13 +1218,15 @@ pub mod pallet {
             #[pallet::compact] loc_id: T::LocId,
             name: <T as pallet::Config>::Hash,
         ) -> DispatchResultWithPostInfo {
-            let who = T::IsLegalOfficer::ensure_origin(origin.clone())?;
+            let who = ensure_signed(origin)?;
 
             if !<LocMap<T>>::contains_key(&loc_id) {
                 Err(Error::<T>::NotFound)?
             } else {
                 let loc = <LocMap<T>>::get(&loc_id).unwrap();
-                if loc.owner != who {
+                let ack_by_owner = loc.owner == who;
+                let ack_by_verified_issuer = Self::verified_issuers_by_loc(loc_id, &who).is_some();
```

Comment:
`verified_issuers_by_loc` should probably be renamed into (for example) `selected_verified_issuer` as it returns a single result given LOC ID and verified issuer account ID.